php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2130,6 +2147,55 @@ abstract class NDB_BVL_Instrument extends NDB_Page
         return $dob;
     }
 
+    /**
+     * Get the candidates DOD.
+     *
+     * @return ?string The candidate's DOD.
+     */
+    function getDoD(): ?string
```

Comment:
Since this is a date that should really never change, why not return a ?\DateTimeImmutable?

Reply:
I agree that's my favourite solution but not necessary.